File: src/test/java/it/xpug/kata/birthday_greetings/EmployeeTest.kt
package it.xpug.kata.birthday_greetings

import org.assertj.core.api.Assertions
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class EmployeeTest {
    @Test
    fun testBirthday() {
        val employee = Employee("foo", "bar", "1990/01/31", "a@b.c")

        assertThat(employee.isBirthday(XDate("2008/01/30")))
            .`as`("not his birthday")
            .isFalse()
        assertThat(employee.isBirthday(XDate("2008/01/31")))
            .`as`("his birthday")
            .isTrue()
    }
}

File: csharp/BirthdayGreetings.Tests/EmployeeTest.cs
using NUnit.Framework;
using BirthdayGreetings;

namespace BirthdayGreetings.Tests
{
    [TestFixture]
    public class EmployeeTest
    {
        [Test]
        public void TestBirthday()
        {
            var employee = new Employee("foo", "bar", "1990/01/31", "a@b.c");

            Assert.That(employee.IsBirthday(new XDate("2008/01/30")), Is.False, "not his birthday");
            Assert.That(employee.IsBirthday(new XDate("2008/01/31")), Is.True, "his birthday");
        }
    }
} 